extensions/notebook/src/book/bookTocManager.ts

```diff
@@ -373,11 +380,11 @@ export class BookTocManager implements IBookTocManager {
 			}
 		}
 		else if (element.contextValue === 'savedNotebook') {
+			// the notebook is part of a book so we need to modify its toc as well
+			const findSection = { file: element.book.page?.file?.replace(/\\/g, '/'), title: element.book.page?.title };
```

Comment:
Why did this get moved? Looks like it's only used in the previous location.

Reply:
The name of the file is changed if there is a file with the same name in the target book in the addNotebook method. In the source book toc, I need to  look for the notebook using it's unchanged name in order to remove it.